mkosi.images/system/mkosi.conf

```diff
@@ -4,7 +4,6 @@
 InitrdInclude=initrd/
 
 [Output]
-@Format=directory
```

Comment:
It took me a while to figure out why `podman` started failing with
```
Error: create /etc: not a directory
```
:-)

@DaanDeMeyer I'll change it locally but I think in the context of bisecting there should probably be a way to "freeze" the mkosi stuff to avoid things like that. I haven't figured out how to do it yet though.